content/_data/changelogs/weekly.yml

```diff
@@ -16104,6 +16104,48 @@
   # pull: 6623 (PR title: Bump spotless-maven-plugin from 2.22.5 to 2.22.6)
   # pull: 6624 (PR title: Bump jenkins-test-harness from 1752.v86627a_c48d91 to 1753.v45c760e2400f)
 
+  - version: '2.355'
+    date: 2022-06-13
+    changes:
+      - type: rfe
+        category: rfe
+        pull: 6460
+        issue: 68373
+        authors:
+          - janfaracik
+          - timja
+        pr_title: "[JENKINS-68373] Update form validation appearance"
+        message: |-
+          Update form validation appearance.
+      - type: bug
+        category: bug
+        pull: 6650
+        issue: 68735
+        authors:
+          - Langer0416
+        pr_title: "[JENKINS-68735] Missing sidepanel for user configuration if password\
+          \ is empty"
+        message: |-
+          Add sidepanel if users enter an empty password or it differs between the current and the new one.
+      - type: bug
+        category: bug
+        pull: 6650
+        issue: 68735
+        authors:
+          - Langer0416
+        pr_title: "[JENKINS-68735] Missing sidepanel for user configuration if password\
+          \ is empty"
+        message: |-
+          Add sidepanel if users enter an empty password or it differs between the current and the new one.
```

Comment:
[#6654](https://github.com/jenkinsci/jenkins/pull/6654) reverted this change.  It needs to be removed before the changelog is published.
```suggestion
```